Packs/TaniumThreatResponse/Integrations/TaniumThreatResponseV2/TaniumThreatResponseV2.py

```diff
@@ -891,11 +891,19 @@ def delete_intel_doc(client, data_args):
     params = {
         'id': data_args.get('intel_doc_id')
     }
-    raw_response = client.do_request('DELETE', '/plugin/products/'
-                                               f'{client.get_threat_response_endpoint()}'
-                                               f'/api/v1/intels/', params=params)
+    try:
+        raw_response = client.do_request('DELETE', '/plugin/products/'
+                                                   f'{client.get_threat_response_endpoint()}'
+                                                   f'/api/v1/intels/', params=params)
+
+    # If the user provided a intel doc ID which does not exist, the do_request will throw HTTPError exception
+    # with a "Not Found" message.
+    except requests.HTTPError as e:
+        if 'not found' in str(e):
+            raise DemistoException(f'Check the intel doc ID and try again.\n({str(e)})')
```

Comment:
```suggestion
            raise DemistoException(f'The intel doc ID does not exist\n({str(e)})')
```

shouldn't it be something like this?

Reply:
```suggestion
            raise DemistoException(f'Could not find the intel doc ID.\n({str(e)})')
```